addons/mail/static/src/core/web/mail_composer_chatgpt.js

```diff
@@ -1,43 +1,64 @@
-import { ChatGPTPromptDialog } from "@html_editor/main/chatgpt/chatgpt_prompt_dialog";
-
-import { htmlJoin } from "@mail/utils/common/html";
-
-import { Component, markup } from "@odoo/owl";
-
 import { _t } from "@web/core/l10n/translation";
 import { registry } from "@web/core/registry";
 import { standardFieldProps } from "@web/views/fields/standard_field_props";
 
+import { htmlJoin } from "@mail/utils/common/html";
+
+import { Component, markup, onMounted, onWillUnmount } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+
 export class MailComposerChatGPT extends Component {
     static template = "mail.MailComposerChatGPT";
     static props = { ...standardFieldProps };
 
     setup() {
         this.btnLabel = _t("AI"); // workaround to translate short string
+        this.store = useService("mail.store");
+        this.orm = useService("orm");
+        this.areWeInDiscuss = this.store.discuss.isActive
+        onMounted(() => {
+            this.store["mail.message"].insertButtonCaller = this.props.record.id
+            this.store.discuss.isActive = false;
+        });
+        onWillUnmount(() => {
+            this.store["mail.message"].insertButtonCaller = false
+            this.store.discuss.isActive = this.areWeInDiscuss;
```

Comment:
Hello, I don't know what we are trying to do but `discuss.isActive` should definitively not be changed in any of this code.